### PR TITLE
Registry integration tests

### PR DESCRIPTION
**Pull request by smartcontrart**

### CODE REVIEW

File: source/registry/package.json [hunk 1]
```diff
@@ -20,7 +20,6 @@
     "typechain": "tsc -b",
     "coverage": "hardhat coverage",
     "test": "hardhat test",
-    "test:unit": "hardhat test ./test/unit.js",
```
Comment (dmosites) [inline source/registry/package.json:23]: `test:unit` is called by CI so let's just update the file name here to test/Registry.js.

Comment (smartcontrart) [reply source/registry/package.json:23]: That won't run test/RegistryIntegration.js. Is that ok?

Comment (dmosites) [reply source/registry/package.json:23]: Good point, updated here: https://github.com/airswap/airswap-protocols/pull/1176

Comment (smartcontrart) [reply source/registry/package.json:23]: @dmosites I believe this is not resolved. Correct? `"test": "hardhat test",` will run both Registry.js and RegistryIntegration.js.

Let me know if there's anything else to be done on that.

**Review by dmosites — APPROVED** [review]

**Pull request merged**